.github/workflows/sonarcloud.yml

```
@@ -29,27 +29,26 @@ jobs:
+            -Dsonar.projectKey=Learnathon-By-Geeky-Solutions_ode-to-code
             -Dsonar.organization=learnathon-by-geeky-solutions
             -Dsonar.java.version=17
             -Dsonar.verbose=true
             -Dsonar.javascript.lcov.reportPaths=coverage/lcov.info
             -Dsonar.sources=lib
-            -Dsonar.exclusions=**/*.g.dart,**/*.freezed.dart,**/generated/**,**/__generated__/**,**/mocks.dart,**/*.css
-        env:
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+            -Dsonar.tests=test
+            -Dsonar.test.inclusions=test/**/*_test.dart
+            -Dsonar.exclusions=**/*.g.dart,**/*.freezed.dart,**/generated/**,**/__generated__/**,**/mocks.dart,**/*.css
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)